lib/mutant/mutator/node/define.rb

```diff
@@ -25,6 +25,20 @@ class Instance < self
 
           children :name, :arguments, :body
 
+          def dispatch
+            super()
+            emit_remove_method
+          end
+
+          # Remove an instance method
+          #
+          # @return [undefined]
+          #
+          # @api private
+          #
```

Comment:
We do not place a newline between the `@api private` and the method anymore. See https://github.com/mbj/mutant/pull/364/files